MonMaraicher/DetailFarmerView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DetailFarmerView.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 27/02/2024.
+//
+
+import SwiftUI
+import MapKit
+
+struct DetailFarmerView: View {
+    
+    @State var selectedFarmer: FarmerPlace?
+    
+    @Binding var show: Bool
```

Comment:
Same here, why making that internal and not private?

Reply:
Why do you need that property/binding? I don't get it

Reply:
I removed this show property